pkg/config/linters_settings.go

```diff
@@ -486,7 +486,8 @@ type NoLintLintSettings struct {
 }
 
 type NoNamedReturnsSettings struct {
-	AllowErrorInDefer bool `mapstructure:"allow-error-in-defer"`
+	AllowErrorInDefer  bool `mapstructure:"allow-error-in-defer"` // Deprecated: use ReportErrorInDefer instead.
```

Comment:
Let's remove `allow-error-in-defer` because there is no any release of golangci-lint with this setting.